src/core/vertexmodel.h

```diff
@@ -49,6 +49,9 @@ class QFIELD_CORE_EXPORT VertexModel : public QAbstractListModel
     Q_PROPERTY( int ringCount READ ringCount NOTIFY ringCountChanged )
     //! determines if the model has changes
     Q_PROPERTY( bool dirty READ dirty NOTIFY dirtyChanged )
+
+    Q_PROPERTY( bool angleFonud READ angleFonud NOTIFY angleFonudChanged )
```

Comment:
As discussed, this should be an int property called snappedAngle.

Also, don't forget documentation.